script.js

```diff
@@ -1,4 +1,171 @@
+    return player2Result;
+  }
+};
+//Determine order for player 2
+var getPlayer2Order = function (input) {
+  var player2OrderResult = "";
+  var player2Order = input;
+  if (currentGameMode == "Pending player 2 order") {
+    if (player2Order == 1) {
+      player2OrderResult = `Player 2<br><br> You have selected ${player2Dice}. <br><br> The number you have is ${player2Dice[0]}${player2Dice[1]}.<br><br> You may click submit to find out the winner.`;
+      currentGameMode = "Determine winner";
+      player2DiceResult.push(player2Dice[0], player2Dice[1]);
+      console.log(player2DiceResult);
+    } else if (player2Order == 2) {
+      player2OrderResult = `Player 2<br><br> You have selected ${player2Dice}. <br><br> The number you have is ${player2Dice[1]}${player2Dice[0]}.<br><br> You may click submit to find out the winner.`;
+      currentGameMode = "Determine winner";
+      player2DiceResult.push(player2Dice[1], player2Dice[0]);
+      console.log(player2DiceResult);
+    } else {
+      player2OrderResult = `You have keyed in an invalid response. The numbers you have are ${player2Dice}. <br><br> To choose the order of numbers, please key in: <br><br> 1 for Dice 1 to be first. <br><br> 2 for Dice 2 to be first.`;
+    }
+    console.log(main, "Player 2 order ok");
+    return player2OrderResult;
+  }
+};
+//Get winner outcome
+var getWinnerOutcome = function () {
+  if (player1DiceResult[0] > player2DiceResult[0]) {
+    winnerOutcome = `Congratulations Player 1! <br><br> You have won! <br><br> Player 1 had selected ${player1DiceResult[0]}${player1DiceResult[1]}.<br><br> Player 2 had selected ${player2DiceResult[0]}${player2DiceResult[1]}.`;
+    console.log("winner criteria 1");
+  } else if (player1DiceResult[0] < player2DiceResult[0]) {
+    winnerOutcome = `Congratulations Player 2! <br><br> You have won! <br><br> Player 2 had selected ${player2DiceResult[0]}${player2DiceResult[1]}.<br><br> Player 1 had selected ${player1DiceResult[0]}${player1DiceResult[1]}.`;
+    console.log("winner criteria 2");
+  } else if (player1DiceResult[1] > player2DiceResult[1]) {
+    winnerOutcome = `Congratulations Player 1! <br><br> You have won! <br><br> Player 1 had selected ${player1DiceResult[0]}${player1DiceResult[1]}.<br><br> Player 2 had selected ${player2DiceResult[0]}${player2DiceResult[1]}.`;
+    console.log("winner criteria 3");
+  } else if (player2DiceResult[1] > player1DiceResult[1]) {
+    winnerOutcome = `Congratulations Player 2! <br><br> You have won! <br><br> Player 2 had selected ${player2DiceResult[0]}${player2DiceResult[1]}.<br><br> Player 1 had selected ${player1DiceResult[0]}${player1DiceResult[1]}.`;
+  } else {
+    winnerOutcome = `It's a draw! <br><br> Player 1 had selected ${player1DiceResult[0]}${player1DiceResult[1]}.<br><br> Player 2 had selected ${player2DiceResult[0]}${player2DiceResult[1]}.`;
+  }
+  return winnerOutcome;
+};
+
 var main = function (input) {
```

Comment:
Main function is used perfectly! I can't ask for more! Maybe.. just maybe.. if you wanted to make main even cleaner.. you could shift the entire block into a ```gameStart()``` function.  :)